flow-typed/types/kava-config.js

```diff
@@ -33,7 +33,8 @@ type _KavaConfigObject = {
   customVar3?: Object,
   applicationVersion?: string,
   playbackContext?: string,
-  tamperAnalyticsHandler?: Function
+  tamperAnalyticsHandler?: Function,
+  userId: string
```

Comment:
let's move it next to `playbackContext` and move `tamperAnalyticsHandler?: Function,` below `serviceUrl?: string,
`
Also, user id is optional so add `?`